frontend/src/app/conference/conference-detail/conference-detail.component.ts

```diff
@@ -27,11 +39,26 @@ export class ConferenceDetailComponent implements OnInit {
   }
 
   interested(): void {
-    return;
+    this.userService.getPCMemberByUsername(this.authService.currentUser)
+      .subscribe(user => this.conference.interested.push(user));
+    this.conferenceService.updateConference(this.conference);
+    this.dialog.open(InterestedDialogComponent,
+      {width: '300px'})
+      .afterClosed().subscribe();
   }
 
   private getConference(): void {
     const id = +this.route.snapshot.paramMap.get('id');
     this.conferenceService.getConference(id).subscribe(conference => this.conference = conference);
   }
+
+  isInterested(): boolean {
```

Comment:
interested is not a self explanatory name. "canBeInterested" on the other hand should be good